l1-contracts/contracts/common/L1ContractErrors.sol

```diff
@@ -51,6 +51,10 @@ error DiamondAlreadyFrozen();
 error DiamondFreezeIncorrectState();
 // 0xa7151b9a
 error DiamondNotFrozen();
+//
```

Comment:
how do you compute the hex?

Reply:
But if the question is whether we have some automated tooling for that, then no, not atm

Reply:
thanks! updated